templates/core/terraform/servicebus.tf

```diff
@@ -9,17 +9,15 @@ resource "azurerm_servicebus_namespace" "sb" {
 }
 
 resource "azurerm_servicebus_queue" "workspacequeue" {
-  name                = "workspacequeue"
-  resource_group_name = azurerm_resource_group.core.name
```

Comment:
rg name is optional  and redundant?

Reply:
namespace_id includes all the info the resource needs.